server/konnectors/edf.coffee

```diff
@@ -0,0 +1,1347 @@
+xml2js = require 'xml2js'
+request = require 'request'
+async = require 'async'
+moment = require 'moment'
+cozydb = require 'cozydb'
+File = require '../models/file'
+
+parser = new xml2js.Parser()
+builder = new xml2js.Builder headless: true
+
+# TODO :
+# Move updateOrCreate in a lib
+# multi account
+# multi contract
+
+# Models
+Client = cozydb.getModel 'Client',
+    clientId: String
+    vendor: String
+    numeroAcc: String
+    address: Object
+    name: Object
+    email: String
+    telMobile: String
```

Comment:
Some fields here and below are named in French, others are named in English (typically `commercialContract` vers `telMobile` or `numeroDepannage`).